src/components/Header/Header.jsx

```diff
@@ -91,7 +91,7 @@ export function Header(props) {
                         }
                     </div>
                     <div className="flex-column items-center mb-5">
-                        <h1 className={`text-lg text-center + ${styles.headerText}`}>Busque pelo código da estação</h1>
+                        <h1 className={`text-lg text-center + ${styles.headerText}`}>Saiba quais linhas de ônibus passam nos pontos da cidade</h1>
```

Comment:
```suggestion
                        <h1 className={`text-lg text-center + ${styles.headerText}`}>Saiba quais linhas passam nos pontos da cidade</h1>
```

só tirar o onibus por conta do BRT, @Gabriel0109 vale mudar tbm na home